packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -7873,6 +7873,7 @@ void main() {
     await tester.sendKeyEvent(LogicalKeyboardKey.arrowRight);
     await tester.pump();
     expect(focusNode3.hasPrimaryFocus, isTrue);
+  // ignore: deprecated_member_use
```

Comment:
Here and everywhere else these ignores where added: What's the plan for when the deprecated member is removed? Delete all these tests? Migrate them? (per style guide there should be a comment on these, but given how wide-spread this is that's maybe a bit excessive? Still, would be good to document somewhere what the expected path forward is when the deprecated member is removed)

Reply:
Added TODOs. The path forward is to just remove the variant and keep the test running for the new system only.